tests/conftest.py

```diff
@@ -364,10 +367,23 @@ def _common_wheel_editable_install(
     wheel_candidates = list(common_wheels.glob(f"{package}-*.whl"))
     assert len(wheel_candidates) == 1, wheel_candidates
     install_dir = tmpdir_factory.mktemp(package) / "install"
-    Wheel(wheel_candidates[0]).install_as_egg(install_dir)
-    (install_dir / "EGG-INFO").rename(install_dir / f"{package}.egg-info")
-    assert compileall.compile_dir(str(install_dir), quiet=1)
-    return install_dir
+    lib_install_dir = install_dir / "lib"
+    bin_install_dir = install_dir / "bin"
+    with zipfile.ZipFile(wheel_candidates[0]) as zf:
+        install(
+            WheelFile(zf),
```

Comment:
```suggestion
    with WheelFile.open(wheel_candidates[0]) as source:
        install(
            source,
```

Reply:
Yeah, that's me reading half of the docs only ;)